person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parental_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parental_permission = parental_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
+  end
```

Comment:
- [OPTIONAL] You don't have to write `? true : false` this part ( you can remove it), you can just write ` of_age? || @parent_permission ` because `||` automatically it will return true if one of them true 👌